app/brave_main_delegate.cc

```diff
@@ -217,10 +217,6 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
 #endif
   };
 
-  if (chrome::GetChannel() == version_info::Channel::CANARY) {
```

Comment:
we probably can now erase `#include "chrome/common/channel_info.h"`?

Reply:
Good catch, also removed `#include "brave/common/brave_features.h"` and `#include "components/version_info/channel.h"`